test/molecule-role/molecule/vms/create.yml

```diff
@@ -58,6 +58,10 @@
         from_port: "{{ connection_port_after_start_windows }}"
         to_port: "{{ connection_port_after_start_windows }}"
         cidr_ip: '0.0.0.0/0'
+      - proto: tcp
+        from_port: "{{ connection_port_after_start_fedora_non_broker }}"
```

Comment:
lets drop this as discussed, instead ensure there is traffic over the windows->ubuntu connection